patchwork/logger.py

```diff
@@ -160,3 +160,28 @@ def init_cli_logger(log_level: str) -> logging.Logger:
     setattr(logger, "freeze", th.freeze)
     logger.setLevel(logging.TRACE)
     return logger
+
+def init_debug_logger(log_level: str) -> logging.Logger:
```

Comment:
This function seems duplicated.

Reply:
this function can be modified later on, if we want to make any changes in logging during debugging.

Reply:
If we do need to change it later on, we can add it later.